src/layouts/DocLayout.tsx

```diff
@@ -1,52 +1,68 @@
-import Link from "next/link";
-import { useRouter } from "next/router";
-import type { FC } from "react";
+import Link from 'next/link'
```

Comment:
We can discuss how we're going to finish out the docs, but if we're going to both be writing, then we should add a `.prettierrc` (or similar) file to the project. Otherwise, these changes will keep bouncing back and forth.